### PR TITLE
GDP + CITY (IMP)

### PR DESCRIPTION
**Pull request by therealtabs**
- I added a city option, which increases our GDP 
-Im unsure as to why selecting "Build  a Random Invention" results in a loss of gdp by 5
- I adjusted GDP to be increased by the no of military factories and cities
- See if this works for you, wont be merging this otherwise

### CODE REVIEW

**Comment by HasanAtir** [comment]
Need you to sit me down and run me through what's happening and why. Kinda lost in the code

File: Main2.cpp [hunk 1]
```diff
@@ -152,11 +159,16 @@ int main()
         cout<<pop_grow<<endl;
         food_grow = land*1000;
         mili_grow = military_factory*25;
+        double x = double(city);
+        gdp = ((pow(1.1,x))*gdp); // Growth of GDP with City Growth
+        double y = double(military_factory);
+        gdp = ((pow(1.1,y))*gdp); // Growth of GDP with Military Factory growth
 
-        food_stores = (food_stores - population) + (food_grow);
+        food_stores = (food_stores - population) + (food_grow*city);// Food growth increases with city count
```
Comment (HasanAtir) [inline Main2.cpp:167]: Food isn't grown in cities though, it's grown on country land so I don't thing this should be changed

File: Main2.cpp [hunk 2]
```diff
@@ -203,8 +218,14 @@ int main()
             building_factory = true;        //Boolean that starts construction of factory
             remaining_construction_time = 3;//Estimated time until factory finishes construction
         }
+        else if (next_year_option ==4)
+        {
+            year++;
+            building_city = true;
+            rem_con_city = 5;
```
Comment (HasanAtir) [inline Main2.cpp:225]: Not a proper variable name

File: Main2.cpp [hunk 3]
```diff
@@ -262,6 +286,7 @@ int main()
             else
             {
                 cout<<"You have lost the war!\n\n";
+                gdp = gdp*0.9;
```
Comment (HasanAtir) [inline Main2.cpp:289]: gdp has to be gdp/capita. So cities should increase commerce rather than gdp directly

Comment (therealtabs) [reply Main2.cpp:289]: Thats what i figured too this morning, ill fix this up and send in a new pull

File: Main2.cpp (hunk 2 above)
Comment (therealtabs) [reply Main2.cpp:225]: Ill edit it

File: Main2.cpp (hunk 1 above)
Comment (therealtabs) [reply Main2.cpp:167]: But since cities arent actually taking up land space, we can have it as way to increment food stock nai?

**Comment by therealtabs** [comment]
I'm gonna edit the code a little and send in a new pull with comments, then if issue arises i'll run you through

**Pull request closed without merging**